src/main/java/ch/zhaw/psit4/data/jpa/entities/Dial.java

```diff
@@ -0,0 +1,59 @@
+package ch.zhaw.psit4.data.jpa.entities;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+/**
+ * Action dial has one or more dialplan and sipclient
+ * Created by beni on 18.04.17.
+ */
+
+@Entity
+public class Dial {
+
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @Column(nullable = false)
+    private String name;
```

Comment:
Dial has no name field, it is not needed.
Add a field "private String priority" for the action.
Add a "private String timeout" field.